TAScheduler/domain/CommandHandler.py

```diff
@@ -19,7 +19,7 @@ def ProcessCommand(self, cmdString: str) -> str:
             return 'Badly formatted command'
 
         handler = {
-            'exit': self._ExitHandler,
```

Comment:
I am fine with this, but does anyone else have any issues with getting rid of the exit command?